exercises/error-handling/error_handling_test.go

```diff
@@ -11,7 +11,7 @@ import (
 // as defined by the expectations of this test suite.
 //
 // Also define a testVersion with a value that matches
-// the internal targetTestVersion here.
+// the targetTestVersion here.
```

Comment:
now this line is different than all the other lines! Everything else still says "internal `targetTestVersion`". Worth making everything consistent?